content/concepts/scaling-deployments/content.md

```diff
@@ -8,14 +8,104 @@ title = "Scaling Deployments"
   sticky = true
 +++
 
-### Overview
+## Overview
 
-TODO
+The most common way to use KEDA is to scale a deployment.  In this model, KEDA acts to monitor the event source and feed that data to Kubernetes and the HPA (Horizontal Pod Autoscaler) to drive rapid scale of a deployment.  Each replica of a deployment is actively pulling items from the event source.  With KEDA and scaling deployments you can scale based on events while also preserving rich connection and processing semantics with the event source (e.g. in-order processing, retries, deadletter, checkpointing).
```

Comment:
This goes "very deep" for people whom are new to KEDA & Kubernetes, maybe we can give them the elevator pitch and then go more in detail?

```suggestion
Deployments are the most common way to scale workloads with KEDA.

It allows you to define the Kubernetes Deployment that you want KEDA to scale based on a scale trigger. KEDA will monitor that service and based on the events that occur it will automatically scale your deployment out/in accordingly.

Behind the scenes, KEDA acts to monitor the event source and feed that data to Kubernetes and the HPA (Horizontal Pod Autoscaler) to drive rapid scale of a deployment.  Each replica of a deployment is actively pulling items from the event source.  With KEDA and scaling deployments you can scale based on events while also preserving rich connection and processing semantics with the event source (e.g. in-order processing, retries, deadletter, checkpointing).
```

How about this?